spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+# spec/controllers/users_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get :index
+      expect(response).to be_successful
+    end
+
+    it 'renders the index template' do
+      get :index
```

Comment:
- [ ] Just as I pointed out above, You are meant to check if the correct placeholder text was rendered, you need to add placeholder text in the `users index and show` view file and then follow the direction I suggested above to see if the body includes the placeholder. I'll suggest you implement these accordingly. :smile: :wave: